octo/compute-version.sh

```diff
@@ -10,6 +10,11 @@ else
   VERSION=$(git rev-parse --short HEAD)
 fi
 
+MAJOR_VERSION="$(echo "${VERSION}" | awk -F '.' '{print $1 }')"
+MINOR_VERSION="$(echo "${VERSION}" | awk -F '.' '{print $1 "." $2 }')"
```

Comment:
I think this assumes that we'll take the first branch of the previous if statement. The first branch happens when we cut a new release of something. You end up with a `GITHUB_REF='refs/tags/v8.8.0'`. 

The `GITHUB_REF` would not follow that pattern when it runs on tests. Like which takes the second branch:

```
GITHUB_REF='refs/heads/main' GITHUB_SHA='foo' ./octo/compute-version.sh
::set-output name=version_major::main
::set-output name=version_minor::main.
::set-output name=version::main
```

or one that takes the third branch:

```
GITHUB_REF='refs/pull/186/merge' GITHUB_SHA='foo' ./octo/compute-version.sh
::set-output name=version_major::57dc06f
::set-output name=version_minor::57dc06f.
::set-output name=version::57dc06f
```

Can you change this so that it doesn't output a major/minor version unless we get a 3-digit semver format version?